### ISSUE TITLE
update docs to use the fork

### ISSUE THREAD

**Issue by aladdin-add** [weiran-zsd/dts-cli#9]

**Comment by SidRathi47** [comment]
@aladdin-add How to use this fork?

**Comment by aladdin-add** [comment]
if you are moving from an existing package:
```bash
$ npm install @weiran.zsd/tsdx -D   # for npm users
$ yarn add @weiran.zsd/tsdx -D      # for yarn users
$ pnpm install @weiran.zsd/tsdx -D  # for pnpm users
```

or create a new package:
```bash
npx @weiran.zsd/tsdx create mylib
```

**Comment by SidRathi47** [comment]
Thanks the second option would be great to have in the docs. I think we might need to modify the docs a little.

**Comment by SidRathi47** [comment]
@aladdin-add Just checked, looks like Rollup breaks here too, 
```
$ parcel index.html
Server running at http://localhost:1234 
🚨  /home/sidwebworks/Desktop/Github/mdx-playground/example/index.tsx: Invalid Version: undefined
    at new SemVer (/home/sidwebworks/Desktop/Github/mdx-playground/example/node_modules/semver/semver.js:314:11)

```

**Comment by SidRathi47** [comment]
I think changing it to, `parcel-bundler` fixes the issue, altho HMR kinda breaks

**Comment by aladdin-add** [comment]
closing as fixed in 78370ec4fe56b8319e991d178972eaec19b16226
